tests/functional/c/consider/consider_using_str_partition.txt

```diff
@@ -0,0 +1,21 @@
+consider-using-str-partition:7:12::Consider using str.partition()
```

Comment:
```suggestion
consider-using-str-partition:7:12::Consider using 'SEQ.partition(',')[0]' instead.
```
Would be nice to have imho :)

Reply:
I will look into how this can be done

Reply:
> Would be nice to have imho :)

Added in b2fd53e3a33a5f8631c4842c365c595b0c86b3b5

Reply:
Very nice result ! I only wonder if regex is the best (faster) way to do it as we have easy access to the AST with `node`.